client/sites/components/guides/sites-guide.tsx

```diff
@@ -0,0 +1,75 @@
+import { Guide } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import guideStep1 from 'calypso/assets/images/sites/sites-guide-1.png';
+import guideStep2 from 'calypso/assets/images/sites/sites-guide-2.png';
+import './sites-guide.scss';
+import { useDispatch, useSelector } from 'calypso/state';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getPreference } from 'calypso/state/preferences/selectors';
+
+const PREFERENCE_KEY = `sites_guide_is_dismissed`;
+
+const SitesGuide = () => {
+	const [ isOpen, setIsOpen ] = useState( true );
+	const [ isImageLoading, setIsImageLoading ] = useState( true );
+	const dispatch = useDispatch();
+	const isDismissed = useSelector( ( state ) => getPreference( state, PREFERENCE_KEY ) );
+
+	if ( ! isOpen || isDismissed ) {
+		return null;
+	}
+
+	const dismiss = () => {
+		setIsOpen( false );
+		dispatch( savePreference( PREFERENCE_KEY, true ) );
+	};
+
+	return (
+		<Guide
+			className={ `sites-guide ${ isImageLoading ? 'is-loading' : '' }` }
```

Comment:
I'd switch this to `clsx` to be more consistent with the codebase.

Reply:
Updated